library/Ivoz/Provider/Domain/Model/User/UserRepository.php

```diff
@@ -70,5 +70,5 @@ public function count(array $criteria): int;
     /**
      * @return UserInterface[]
      */
-    public function findLatestAddedByCompany(int $companyId): array;
+    public function findLatestAddedByCompany(int $companyId, int $intemNum = 5): array;
```

Comment:
```suggestion
    public function findLatestAddedByCompany(int $companyId): array;
```
This method is only used in the dashboard so we don't need to add any extra parameter